zebra-chain/src/block.rs

```diff
@@ -43,7 +43,10 @@ pub use arbitrary::LedgerState;
 
 /// A Zcash block, containing a header and a list of transactions.
 #[derive(Clone, Debug, Eq, PartialEq)]
-#[cfg_attr(any(test, feature = "proptest-impl"), derive(Serialize))]
+#[cfg_attr(
+    any(test, feature = "proptest-impl", feature = "elasticsearch"),
```

Comment:
Every crate that uses the `elasticsearch` feature needs to have it declared in its `Cargo.toml`, and it needs to activate that feature in the crates that it depends on. (If they have that feature.)

You can use `getblocktemplate-rpcs` as an example:
https://github.com/ZcashFoundation/zebra/blob/2e379811c3a8f4dd74c8e0e735738dcbe52d4288/zebrad/Cargo.toml#L28-L35

The list of activated dependencies or features can be empty if there's nothing extra needed.
https://github.com/ZcashFoundation/zebra/blob/2e379811c3a8f4dd74c8e0e735738dcbe52d4288/zebra-chain/Cargo.toml#L20-L23

Reply:
Yea, thats needed. And some other changes. They were done [here](https://github.com/ZcashFoundation/zebra/pull/6709/commits/a809644bd9ac088af681412143a0d560352243de)

This command is now worrking:

```
cargo install zebrad --features="elasticsearch" --git https://github.com/ZcashFoundation/zebra --branch es-serialize-buld-error zebrad
```

I am moving this PR out of draft.

Reply:
We're currently in a release freeze, so I'm going to mark this PR to merge after the release.